tap_twitter_ads/streams.py

```diff
@@ -12,537 +12,1592 @@
-        'data_key': 'data',
-        'key_properties': ['id'],
-        'replication_method': 'INCREMENTAL',
-        'replication_keys': ['updated_at'],
-        'params': {
-            'sort_by': ['updated_at-desc'],
-            'with_deleted': '{with_deleted}',
-            'count': 1000,
-            'cursor': None
-        }
-    },
-    # Reference: https://developer.twitter.com/en/docs/ads/campaign-management/api-reference/content-categories#content-categories
-    'content_categories': {
-        'path': 'iab_categories',
-        'data_key': 'data',
-        'key_properties': ['id'],
-        'replication_method': 'FULL_TABLE',
-        'params': {
-            'count': 1000,
-            'cursor': None
-        }
-    },
-    # Reference: https://developer.twitter.com/en/docs/ads/campaign-management/api-reference/funding-instruments#funding-instruments
-    'funding_instruments': {
-        'path': 'accounts/{account_id}/funding_instruments',
-        'data_key': 'data',
-        'key_properties': ['id'],
-        'replication_method': 'INCREMENTAL',
-        'replication_keys': ['updated_at'],
-        'params': {
-            'sort_by': ['updated_at-desc'],
-            'with_deleted': '{with_deleted}',
-            'count': 1000,
-            'cursor': None
-        }
-    },
-    # Reference: https://developer.twitter.com/en/docs/ads/campaign-management/api-reference/iab-categories#iab-categories
-    'iab_categories': {
-        'path': 'content_categories',
-        'data_key': 'data',
-        'key_properties': ['id'],
-        'replication_method': 'FULL_TABLE',
-        'params': {}
-    },
-    # Reference: https://developer.twitter.com/en/docs/ads/campaign-management/api-reference/line-items#line-items
-    'line_items': {
-        'path': 'accounts/{account_id}/line_items',
-        'data_key': 'data',
-        'key_properties': ['id'],
-        'replication_method': 'INCREMENTAL',
-        'replication_keys': ['updated_at'],
-        'params': {
-            'sort_by': ['updated_at-desc'],
-            'with_deleted': '{with_deleted}',
-            'count': 1000,
-            'cursor': None
-        },
-        'children': {
-            # Reference: https://developer.twitter.com/en/docs/ads/campaign-management/api-reference/targeting-criteria#targeting-criteria
-            'targeting_criteria': {
-                'path': 'accounts/{account_id}/targeting_criteria',
-                'data_key': 'data',
-                'key_properties': ['line_item_id', 'id'],
-                'replication_method': 'FULL_TABLE',
-                'parent_ids_limit': 200,
-                'params': {
-                    'line_item_ids': '{parent_ids}', # up to 200 comma delim ids
-                    'with_deleted': '{with_deleted}',
-                    'count': 1000,
-                    'cursor': None
+LOGGER = singer.get_logger()
+
+def update_currently_syncing(state, stream_name):
```

Comment:
Add function comments for each function.

Reply:
Added comments for all functions